app/utils.ts

```diff
@@ -0,0 +1,12 @@
+import * as bigNumber from "bignumber.js";
+import { commitmentStartDate } from "./config";
+import { asEtherNumber } from "./web3/utils";
+
+export async function loadIcoParamsFromEnviroment() {
+  return {
+    minCap: asEtherNumber(new bigNumber(10 * 10 ** 18)),
+    maxCap: asEtherNumber(new bigNumber(666 * 10 ** 18)),
+    startDate: commitmentStartDate,
+    endDate: "2017-11-15",
+  };
+}
```

Comment:
These numbers can change, actually i think if we go forward with the two before ICO states they might not be needed at all